api/v1/verticadb_types.go

```diff
@@ -1029,6 +1029,15 @@ type VerticaDBStatus struct {
 	// +optional
 	// The list of current TLS configurations for different components
 	TLSConfigs []TLSConfigStatus `json:"tlsConfigs,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=status
+	// +optional
+	// The secret name that contains the current password for the
+	// database's superuser. The secret must have a key named password. To
+	// store this secret outside of Kubernetes, you can use a secret path
+	// reference prefix, such as gsm://. Everything after the prefix is the name
+	// of the secret in the service you are storing.
```

Comment:
```suggestion
	// The secret name that contains the current password for the
	// database's superuser.
```
This is the status, we don't need all the details from the spec